db-auto-backup.py

```diff
@@ -194,6 +194,9 @@ def backup(now: datetime) -> None:
             data="\n".join(backed_up_containers),
         ).raise_for_status()
 
+    if uptime_kuma_url := os.environ.get("UPTIME_KUMA_URL"):
+        requests.get(uptime_kuma_url)      
```

Comment:
Issue: It's worth adding `raise_for_status` here so that it shows in the event of a network error.

```suggestion
        requests.get(uptime_kuma_url).raise_for_status()      
```